zip-frost.rst

```diff
@@ -0,0 +1,465 @@
+::
+
+  ZIP: Unassigned {numbers are assigned by ZIP editors}
+  Title: FROST for Spend Authorization Signatures
+  Owners: Conrado Gouvea <conrado@zfnd.org>
+          Chelsea Komlo <ckomlo@uwaterloo.ca>
+          Deirdre Connolly <deirdre@zfnd.org>
+  Credits: First Credited
+           ...
+  Status: Draft
+  Category: Wallet
+  Created: 2022-08-dd
+  License: {usually MIT}
```

Comment:
Please specify the license (e.g. `  License: MIT`).

Reply:
Done in https://github.com/zcash/zips/pull/662